### PR TITLE
ABCU Marker usability improvements

### PR DESCRIPTION
**Pull request by kbsmilk**
## About the PR 
ABCU marker now uses pixelaction instead of afterattack, and its range was uncapped. It now makes 2 different noises when selecting/unselecting a tile.

## Why's this needed? 
Marker is less annoying to use now. Previously, it couldn't target wall corners from inside

## Changelog 

```changelog
(u)Smilg
(+)Increased ABCU marker selection range
```

### CODE REVIEW

File: code/WorkInProgress/blueprints.dm [hunk 1]
```diff
@@ -436,8 +436,8 @@
 
 	var/static/savefile/save = new/savefile("data/blueprints.dat")
 
-	afterattack(atom/target as mob|obj|turf, mob/user as mob)
-		if(GET_DIST(src,target) > 2) return
+	pixelaction(atom/target as mob|obj|turf, params, mob/user as mob)
+	//	if(GET_DIST(src,target) > 2) return
 
 		if(!isturf(target)) target = get_turf(target)
```
Comment (Mister-Moriarty) [inline code/WorkInProgress/blueprints.dm:442]:
```suggestion
	pixelaction(atom/target, params, mob/user)
		if(!isturf(target)) target = get_turf(target)
```
Remove the commented out code; Git history will remember.
I don't believe that there is any need for `as mob|obj|turf` here.

**Review by ZeWaka — APPROVED** [review]

File: code/WorkInProgress/blueprints.dm [hunk 2]
```diff
@@ -436,9 +436,7 @@
 
 	var/static/savefile/save = new/savefile("data/blueprints.dat")
 
-	afterattack(atom/target as mob|obj|turf, mob/user as mob)
-		if(GET_DIST(src,target) > 2) return
```
Comment (TheTyrant25) [inline code/WorkInProgress/blueprints.dm:440]: seems you removed the distance check, is that intended?

**Review by TheTyrant25 — APPROVED** [review]
looks good.

File: code/WorkInProgress/blueprints.dm (hunk 2 above)
Comment (Tarmunora) [reply code/WorkInProgress/blueprints.dm:440]: should probably retain *some* distance check (7 or 10), to lock it to stuff on-screen (camera shenanigans etc.)

Comment (kbsmilk) [reply code/WorkInProgress/blueprints.dm:440]: I just copied Floor and Wall Designer's design (obj/item/room_planner), which also has no distance check, and seemingly no problems

Comment (Tarmunora) [reply code/WorkInProgress/blueprints.dm:440]: those should be limited to on-screen too 😱

**Pull request closed without merging**